ultimarc/devices/mini_pac.py

```diff
@@ -349,14 +349,14 @@ def _create_message_(self, config_file, cur_device_config=None):
             # 0x40 in the shift position for all pins designated as shift pins in json config
             for pin in config.pins:
                 try:
-                    action_index, alternate_action_index, shift_index = PinMapping[pin.name]
+                    action_index, alternate_action_index, shift_index = PinMapping[pin.product_name]
                     action = pin.action.upper()
                     if action in IPACSeriesMapping:
                         data.bytes[action_index] = IPACSeriesMapping[action]
                     elif action in macro_dict:
                         data.bytes[action_index] = macro_dict[action]
                     else:
-                        _logger.info(_(f'{pin.name} action "{action}" is not a valid value'))
```

Comment:
this name variable shouldn't be changed to product_name.

Reply:
Fixed